packages/wallet-sdk/src/util/web.ts

```diff
@@ -27,8 +34,15 @@ export function closePopup(popup: Window | null) {
   }
 }
 
-/**
- * TODO: consolidate all UI related helper functions,
- * ones making window.xxx() document.yyy() calls.
- * e.g. WLMobileRelayUI, WalletLinkRelay, ...
- */
+async function appendAppInfoQueryParams(url: URL) {
+  const params = {
+    sdkName: LIB_NAME,
+    sdkVersion: LIB_VERSION,
+    origin: window.location.origin,
+    coopIncompatibility: await checkCrossOriginOpenerPolicyCompatibility(),
```

Comment:
should we just pass what policy the site uses? then we can let the popup decide what to do with it. Also this naming is a bit weird. has a double negative. `coopIncompatibility === false` means its in compatible

Reply:
sgsg, pushed an update to make `checkCrossOriginOpenerPolicy` return policy itself, and passing policy to FE